src/oscontainer.py

```diff
@@ -108,7 +112,11 @@ def oscontainer_build(containers_storage, src, ref, image_name_and_tag,
             tls_arg = '--tls-verify=false'
         else:
             tls_arg = '--tls-verify'
-        run_verbose(['podman', rootarg, 'push', tls_arg, image_name_and_tag])
+        if cert_arg != "":
```

Comment:
Instead of duplicating this code, how about setting it up after `parser.parse_args()` is called below?